reference-architecture/osp-cli/haproxy.cfg.template

```diff
@@ -1,63 +0,0 @@
-global
```

Comment:
I'd say that as we remove the haproxy template and the ability to generate haproxy configuration, we should keep at least an haproxy.cfg.example or something similar...